src/OT/glyf/glyf.hh

```diff
@@ -337,6 +338,19 @@ struct glyf_accelerator_t
     return glyph_for_gid (gid).get_extents_without_var_scaled (font, *this, extents);
   }
 
+  bool paint_glyph (hb_font_t *font, hb_codepoint_t gid, hb_paint_funcs_t *funcs, void *data) const
+  {
+    funcs->push_root_transform (data, font);
+
+    funcs->push_clip_glyph (data, gid);
```

Comment:
Shouldn’t there be a plain `paint_glyph()` for uses like this where there is no clipping is needed to avoid the unnecessary overhead of clipping?

Reply:
Such a function would have to do analysis of the paint graph. I don't think we want to do that.

Reply:
May be I’m missing something, but I was thinking of something that would be used when painting COLRv0 or non-COLR tables where we know it is a simple painting with no clipping involved.

Reply:
You're not missing anything. That's certainly doable to add a specialization call for painting a glyph with a solid color. I suggest doing that later if necessary. What I like currently is how few functions we have and that it unifies COLRv0 and non-COLR tables.

Reply:
Or if Matthias wants to add it that sounds good to me.

Reply:
I think the caller can keep track of the current path, do a fill, then add the path to the clip?
Or something like that